pkg/sql/parser/sql.y

```diff
@@ -5948,15 +5946,10 @@ set_csetting_stmt:
   }
 | SET CLUSTER error // SHOW HELP: SET CLUSTER SETTING
 
-// %Help: ALTER TENANT - alter tenant configuration
-// %Category: Group
-// %Text:
-// ALTER TENANT { <tenant_id> | ALL } SET CLUSTER SETTING <var> { TO | = } <value>
-// ALTER TENANT { <tenant_id> | ALL } RESET CLUSTER SETTING <var>
-// %SeeAlso: SET CLUSTER SETTING
```

Comment:
could these still be here so that if we do start generating docs we have it already?